Conexon/Conexon.github.io
# initial commit

updated notes
-------------
- all geojson (point) state files were updated
- the below `client_geoid.json` and `client_sub.geojson` files have all been updated, and need to be tested
- testing should 
  - confirm that selection of substaitons and blockgroups works as expected
  - confirm that when select all, the sum of the numbers matches those in the updated spreadsheet found  here - https://docs.google.com/spreadsheets/d/1tO0rYffTV3Go7cN85gUgxIJ9gil6LfjJb6IvDjRnbbA/edit#gid=165658630 - the `coop_meter_range` worksheet
- only CO and NY state tiles have been updated

list of coops so far
- [ ] AR,Ashley_Chicot_Electric_Cooperative,ACEC - `not enough data for map 2 yet`
- [ ] AR,Farmers_Electric_Cooperative,FARMERS - `not enough data for map 2 yet`
- [ ] AR,First_Electric_Cooperative,FIRST - `index not selecting all bgs`
- [x] AR,Mississippi_County_Electric,MCECI - `index not selecting all bgs`
- [x] AR,North_Arkansas_Electric_Cooperative,NORTHAR - `green block only on both work; any on both fail`
- [x] AR,OzarksGo,OZARKSGO - `multi-state not prepared yet`
- [x] AR,Petit_Jean_Electric_Cooperative,PETITJEAN - `both wrong in both maps`
- [x] AR,South_Central_Arkansas_Electric_Cooperative,SCAEC - `green only on both works; any method on both fails`
- [ ] AR,Woodruff_Electric_Cooperative_Corporation,WECC - `block groups green works, others fail`
- [x] CO,Delta_Montrose_Electric_Association,DMEA
- [ ] CO,Holy_Cross_Energy,HOLYCROSS - `not enough data for map 2 yet`
- [x] CO,Yampa_Valley_Electric_Association,YVEA
- [x] GA,Central_Georgia_Electric_Membership_Corporation,CGEMC - `both numbers don't match`
- [x] GA,Irwin_Electric_Membership_Cooperative,IEMC - `green only matches`
- [x] GA,Middle_Georgia_Electric_Membership_Cooperative,MGEMC
- [x] GA,Satilla_Rural_Electric_Membership_Coorporation,SREMC
- [x] GA,Southern_Rivers_Energy,SRE
- [x] GA,Tri_County_Electric_Membership_Corporation,TRICNTGA
- [x] IL,Corn_Belt_Energy_Corporation,CORNBELT
- [x] IL,Shelby_Electric_Cooperative,SHELBYEC
- [x] KY,Kenergy,KEN
- [x] MD,Southern_Maryland_Electric_Cooperative,SMECO
- [x] MI,Great_Lakes_Energy_Electric_Cooperative,GREATLAKES
- [x] MI,HomeWorks_Tri_County_Electric_Cooperative,HWTCEC
- [x] MI,Midwest_Energy_Communications,MEC
- [x] MI,Presque_Isle_Electric_and_Gas_Cooperative,PIEG
- [x] MI,Thumb_Electric_Cooperative,THUMB
- [x] MO,Black_River_Electric_Cooperative,BREC
- [x] MO,Callaway_Electric_Cooperative,CALLAWAY
- [x] MO,Central_Missouri_Electric_Cooperative,CMEC
- [x] MO,Co_Mo_Electric,COMO
- [x] MO,Consolidated_Electric_Cooperative_Missouri,CONSOLMO
- [x] MO,Howell_Oregon_Electric_Cooperative,HOEC
- [x] MO,New_Mac_Electric_Cooperative,NEWMAC
- [x] MO,Osage_Valley_Electric_Cooperative,OSAGEVLY
- [x] MO,Ozarks_Electric_Cooperative,OZARKS
- [x] MO,Pemiscot_Dunklin,PEMDUNK
- [x] MO,Platte_Clay_Electric_Coop_Inc,PLATTE
- [x] MO,Sac_Osage_Electric_Cooperative,SACOSAGE
- [x] MO,SEMO_Electric_Cooperative,SEMO
- [x] MO,Three_Rivers_Electric_Cooperative,THREERIVERS
- [x] MO,United_Electric_Cooperative,UTD
- [x] MO,West_Central_Electric_Coop_Inc,WESTCEN
- [x] MS,Alcorn_County_Electric_Power_Association,ACEPA
- [x] MS,Central_Electric_Power_Association,CEPA
- [x] MS,Coahoma_Electric_Power_Association,COEPA
- [x] MS,Coast_Electric_Power_Association,COAST
- [x] MS,Delta_Electric_Power_Association,DELTA
- [x] MS,Dixie_Electric_Power_Association,DIXIEEPA
- [x] MS,East_Mississippi_Electric_Power_Association,EMEPA
- [x] MS,Monroe_County_Electric_Power_Association,MONROE
- [x] MS,Natchez_Trace_Electric_Power_Association,NTEPA
- [x] MS,Northcentral_Electric_Power_Association,NCEPA
- [x] MS,Northeast_Mississippi_Electric_Power_Association,NEMS
- [x] MS,Pearl_River_Valley_Electric_Power_Association,PRVEPA
- [x] MS,Prentiss_County_Electric_Power_Association,PCEPA
- [x] MS,Singing_River_Electric_Cooperative,SREC
- [x] MS,Southern_Pine_Electric,SOUPINE
- [x] MS,Tallahatchie_Valley_Electric_Power_Association,TVEPA
- [x] MS,Tippah_Electric_Power_Association,TPEPA
- [x] MS,Tishomingo_County_Electric_Power_Association,TCEPA
- [x] MS,Tombigbee_Electric_Power_Association,TOMBB
- [x] MS,Twin_County_Electric_Power_Association,TWINCOEPA
- [x] MS,Yazoo_Valley_Electric_Power_Association,YAZOO
- [x] NH,New_Hampshire_Electric_Cooperative,NHEC
- [x] NM,Kit_Carson_Electric_Cooperative,KCEC
- [x] NY,Otsego_Electric_Cooperative,OTSEGO
